.jenkins/agent/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM quay.io/openshift/origin-jenkins-agent-base:latest
 RUN curl -LO "https://github.com/operator-framework/operator-sdk/releases/download/v0.19.4/operator-sdk-v0.19.4-x86_64-linux-gnu" && \
     chmod +x operator-sdk-v0.19.4-x86_64-linux-gnu && mv operator-sdk-v0.19.4-x86_64-linux-gnu /usr/local/bin/operator-sdk
```

Comment:
I'm thinking we might be able to remove this entirely to prevent confusion. We have code in the CI ansible to manage operator-sdk (and golang for that matter), so AFAICT this version is never actually used.

Reply:
I agree. If we can keep the installation of dynamic tooling separate of the base image, that should help keep confusion down.

In downstream we need to install the operator-sdk for the bundle generation, so if we stick to that in stf-run-ci, it should make more sense. I believe it's the `generate_bundle.sh` script that gets used across both upstream and downstream. I'm hoping to build some GitHub Actions that will generate bundles and push them to quay.io on merge to main and stable branches.

Then at least we have a path where the streams can more naturally cross.

As you said, it's natural for upstream CI systems to be more targeting the future, and for downstream to handle the stable. If the build system converges at some point, then I think that's a win.

Reply:
@andyvesel Are you comfortable enough with how this works to take on the task of removing this operator-sdk installation and verifying that the jobs still function correctly? I think it's probably trivial, I just want to be sure you are comfortable with how this container is built for use, and checking on the results. Ping me if you decide to do it, I want to point you to an important checkbox in Jenkins for when we are updating the agent container.

Reply:
@csibbitt  I don't see anything non-trivial, so yes, I can figure this out. I will ping you for more details then